docs/How-To/api/README.md

```diff
@@ -1,15 +1,15 @@
 ---
 Title: REST APIs
-Description: A high-level overview of gRPC-gateway REST Routes in farming module.
+Description: A high-level overview of gRPC-gateway REST routes in farming module.
 ---
 
 ## Swagger Documentation
 
-- [Swagger Docs v0.1.3](https://app.swaggerhub.com/apis-docs/gravity-devs/farming/0.1.3)
+- Swagger Cosmos SDK Farming Module [REST and gRPC Gateway docs]](https://app.swaggerhub.com/apis-docs/gravity-devs/farming/0.1.3)
```

Comment:
I think we should remove redundant `]` at the end:
```suggestion
- Swagger Cosmos SDK Farming Module [REST and gRPC Gateway docs](https://app.swaggerhub.com/apis-docs/gravity-devs/farming/0.1.3)
```

Reply:
I didn't notice that. Great catch!

Reply:
Since @barriebyron grant me permission to push commits if necessary, I will apply this suggestion.

Reply:
as always, feel free to commit any changes at any time! it's better together